### PR TITLE
Add FP8 row-wise gemm (tensilelite)

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

File: tensilelite/Tensile/Common.py [hunk 1]
```diff
@@ -1164,7 +1164,8 @@ def getArchitectureName(gfxName):
     "Gradient":                 False,            # =True set globalWriteElements to gradient mode
     "UseBias":                  0,                # =1 support bias vector on M direction, =2 support bias vector on N direction, =3 support bias vector on both M,N direction
     "BiasSrc":                  "D",              # This parameter is used in gradient + bias. Support A, B, D.
-    "UseScaleAB":               False,            # =True use scaleA, scaleB
+    "UseScaleAB":               False,            # =True use scaleA, scaleB, may be deprecated
```
Comment (jichangjichang) [inline tensilelite/Tensile/Common.py]: If ScaleAB and scaleABvec cannot exist at the same time, It is better use same problem argument but just extend its scope. Please don't create new "UseScaleABVec"
UseScaleAB: 0 # not support
UseScaleAB: 1 # legacy scaling factor
UseScaleAB: 2 # scale vector

Comment (KKyang) [reply tensilelite/Tensile/Common.py]: Scalar AB is not a legacy scaling factor. We are still figuring out a way to make them both exist in one kernel but not increase the kernel size too much.

File: tensilelite/Tensile/Tests/common/gemm/fp8.yaml [hunk 2]
```diff
@@ -219,3 +221,62 @@ BenchmarkProblems:
...
+      HighPrecisionAccumulate: True
+      TransposeA: 1
+      TransposeB: 0
+      UseBeta: True
+      Batched: True
+      BiasDataTypeList: [s]
+      UseBias:       1
+      UseE:          False
+      Activation:    True
+      UseScaleABVec: True
```
Comment (jichangjichang) [inline tensilelite/Tensile/Tests/common/gemm/fp8.yaml]: correct this one

**Review by jichangjichang — APPROVED** [review]
lgtm

**Review by vin-huang — APPROVED** [review]

**Pull request merged**